Gemfile.lock

```diff
@@ -120,6 +120,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-22
+  arm64-darwin-23
```

Comment:
Was this change intentional? It seems fine, but wanted to double check.

Reply:
This is just Sonoma, figured I'd keep it in otherwise it's just gonna keep happening